packaging/tuned/microshift-tuned.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=MicroShift TuneD helper
+Before=microshift.service
+After=tuned.service
+Requires=tuned.service
+
+[Service]
+ExecStart=/usr/bin/microshift-tuned
```

Comment:
This script may reboot host. Should we provide a mandatory command line for the script to actually do something? Otherwise, if someone runs it, they can end up rebooting accidentally.

Reply:
You mean that dry run is default and not dry run is protected with `--something`?

Reply:
There's a variable in the config which controls if reboot is allowed. Wouldn't it be slightly confusing if it was defined `False` but our service always had `--reboot-if-allowed`?

Reply:
We could call it `--live-run` or `--execute`.